cpp/src/io/statistics/statistics.cuh

```diff
@@ -59,36 +63,37 @@ struct stats_column_desc {
   column_device_view const* parent_column;  //!< Pointer to parent column; nullptr if not list type
 };
 
-struct string_stats {
-  const char* ptr;  //!< ptr to character data
-  uint32_t length;  //!< length of string
-  __host__ __device__ __forceinline__ volatile string_stats& operator=(
-    const string_view& val) volatile
+template <typename ReturnType, typename InternalType>
+struct t_array_stats {
```

Comment:
Thanks for making this change.

Reply:
I think it turned out great. Thanks for suggesting it!